backend/content/models.py

```diff
@@ -44,9 +52,27 @@ def __str__(self) -> str:
         return self.question
 
 
-# This is used to set the filename to the UUID of the model, in the Image model.
 def set_filename_to_uuid(instance: Any, filename: str) -> str:
-    """Generate a new filename using the model's UUID and keep the original extension."""
+    """
+    Generate a new filename using the model's UUID and keep the original extension.
+
+    Parameters
+    ----------
+    instance : Any
+        The model instance that the file is being attached to.
+    filename : str
```

Comment:
No stress here, @rohityadav2604, but could we get a blank line between parameter entries? I find it a bit more readable :) I'll send the change along, so just for future PRs, and thanks for this great docstring!